futures-util/src/abortable.rs

```diff
@@ -182,4 +182,13 @@ impl AbortHandle {
         self.inner.aborted.store(true, Ordering::Relaxed);
         self.inner.waker.wake();
     }
+
+    /// Checks whether [`AbortHandle::abort`] was *called* on any associated
+    /// [`AbortHandle`]s, which includes all the [`AbortHandle`]s linked with
+    /// the same [`AbortRegistration`].
+    ///
+    /// This operation has a Relaxed ordering.
```

Comment:
It may be preferable to mention the caveat mentioned in the Abortable::is_aborted docs ("This means that it will return `true` even if ...").